lms/static/scripts/frontend_apps/components/dashboard/OrderableActivityTable.tsx

```diff
@@ -3,13 +3,20 @@ import { DataTable } from '@hypothesis/frontend-shared';
 import { useOrderedRows } from '@hypothesis/frontend-shared';
 import type { OrderDirection } from '@hypothesis/frontend-shared/lib/types';
 import { useMemo, useState } from 'preact/hooks';
+import { useLocation } from 'wouter-preact';
 
 export type OrderableActivityTableProps<T> = Pick<
   DataTableProps<T>,
   'emptyMessage' | 'rows' | 'renderItem' | 'loading' | 'title'
 > & {
   columnNames: Partial<Record<keyof T, string>>;
   defaultOrderField: keyof T;
+
+  /**
+   * Allows to define a URL to navigate to when a row is confirmed via
+   * double-click/Enter key press.
+   */
+  hrefOnConfirmRow?: (row: T) => string;
```

Comment:
```suggestion
  navigateOnConfirmRow?: (row: T) => string;
```

I think this makes it a little more obvious what action happens when the row is confirmed. This could be even more verbose like `navigateToURLOnConfirmRow` but that is starting to get hard to read.